dahu/core/app/scripts/templates/views/workspace/actions/disappear.html

```diff
@@ -1,6 +1,2 @@
 <div class="disappear">
```

Comment:
can't we keep that for further implementation such as specific way for disappearing (fading or moving) ?